img/h/hatchinvest.nz.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 133.54 31.987"><path d="M35.429 13.514H23.065v4.46h12.364zM71.31 7.843h4.85v23.658h-4.85zM63.941.472h19.637v4.46H63.94zM113.4 31.501h4.848V.472H113.4zM128.684.472v13.042h-7.546v4.451h7.554v13.528h4.848V.473zM12.364 13.514H0v4.46h12.364zM20.137.472h-4.848V31.5h4.848zM97.614 0c-6.449 0-9.503 3.346-9.503 9.018v5.309h4.848v-5.66c0-2.95 1.697-4.219 4.655-4.219 3.006 0 4.655 1.26 4.655 4.218v1.65h4.843V9.003c0-5.671-3.001-9.004-9.498-9.004zm4.656 23.307c0 2.91-1.649 4.22-4.655 4.22-2.95 0-4.655-1.31-4.655-4.22v-6.11h-4.848v5.724c0 5.671 3.054 9.066 9.503 9.066 6.497 0 9.503-3.395 9.503-9.066v-1.26h-4.848zm-53.068-4.945 2.861-12.461 2.19 9.538h4.672L55.165.47h-6.207l-6.69 27.007V31.5h4.848v-4.121l1.067-4.556h7.757l1.116 4.751V31.5h4.848v-4.022L59.63 18.36zM35.429 20.88h-4.848v10.621h4.848zm0-20.408h-4.848v10.136h4.848z" fill="#231f20"/></svg>
```

Comment:
I'd rather go with this logo:
![](https://raw.githubusercontent.com/phallobst/phallobst/main/resterampe/hatchinvest.nz.svg)
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="250.667" height="250.667" fill-rule="evenodd"><path d="M107.079 8.667h36.458V242h-36.458z" fill="#33e9d5"/><path d="M0 106.608h93.858v33.991H0z" fill="#f09539"/><path d="M157.162 106.608h93.505v33.991h-93.505z" fill="#a286e7"/></svg>
```